Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -572,6 +572,8 @@ void GridFormattingContext::run(Box const& box, LayoutMode, AvailableSpace const
             independent_formatting_context->parent_context_did_dimension_child_root_box();
         if (child_box_state.content_height() > positioned_box.computed_height)
             positioned_box.computed_height = child_box_state.content_height();
+        if (calculate_min_content_height(positioned_box.box, available_space.width) > positioned_box.computed_height)
+            positioned_box.computed_height = calculate_min_content_height(positioned_box.box, available_space.width);
```

Comment:
`calculate_min_content_height` looks pretty expensive. At the very least, let's not perform that operation twice:
```c++
if (auto min_content_height = calculate_min_content_height(positioned_box.box, available_space.width); min_content_height > positioned_box.computed_height)
    positioned_box.computed_height = min_content_height;
```

Reply:
Oh, yes, sure. Thanks!

Reply:
Intrinsic sizes are actually cached, so the cost in basically just a hash lookup. But it's still even nicer to put it in a local like you've done here :)